ui/src/notebooks/context/query.tsx

```diff
@@ -21,6 +21,10 @@ export interface QueryContextType {
   query: (text: string) => Promise<BothResults>
 }
 
+export interface QueryContextType {
```

Comment:
what is the difference between this and the one on line 20?

Reply:
Literally nothing. Good eyes, looks like I added something that shouldn't have been added